src/main/java/org/dbflute/intro/app/logic/core/FlutyFileLogic.java

```diff
@@ -44,4 +46,21 @@ public void writeFile(File textFile, String content) {
             throw new UncheckedIOException("Cannot write the text file: " + textFile, e);
         }
     }
+
+    /**
+     * Open directory by filer. (e.g. finder if mac, explorer if windows)
+     * Use OS command.
+     *
+     * @param clientName dbflute client project name (NotEmpty)
+     */
+    public void openDir(String clientName, File dir) {
+        if (dir.exists()) {
+            try {
+                Desktop desktop = Desktop.getDesktop();
+                desktop.open(dir);
+            } catch (IOException e) {
+                throw new UncheckedIOException("fail to open directory of" + clientName + ", directory path: " + dir.getAbsolutePath(), e);
+            }
+        }
```

Comment:
How would you feel about handling exception of file not exists.

Reply:
Implemented exception handling.

https://github.com/dbflute/dbflute-intro/pull/250/files#diff-d77e2a8aef692367d7df47040d99853a0d15b98ed37b455a42aaa8738e092733